ibis/backends/dask/dispatcher.py

```diff
@@ -0,0 +1,113 @@
+from multipledispatch import Dispatcher
+
+
```

Comment:
TODO item, can consolidate this with the code in pandas and put in a common location (e.g. that's a good pre-cursor PR to bite off here).